lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -683,6 +737,10 @@ impl Client {
         self.file_cache.get(&file_id)
     }
 
+    fn remove_file_by_id(&mut self, file_id: u32) -> Option<FileCacheObject> {
```

Comment:
Again, more docs. Most people reading this will not have the context that you acquired after working on this for several months. Anything you can include to help orient what this part of the code is for will be helpful for the rest of the team.